### PR TITLE
Click element replaces readyElement(element).click

### PR DESCRIPTION
**Pull request by djansen-redhat**

### CODE REVIEW

**Comment by djansen-redhat** [comment]
Depends on #813

File: functional-test/src/main/java/org/zanata/page/AbstractPage.java [hunk 1]
```diff
@@ -435,4 +371,203 @@ private void assertReady(WebElement targetElement) {
...
+        log.info("Closing {} notifications", notifications.size());
+        for (WebElement notification : notifications) {
+            try {
+                notification.click();
+            } catch (WebDriverException exc) {
+                log.info("Missed a notification X click");
+            }
+        }
+        // Finally, forcibly un-is-active the message container - for speed
+        String script = "return (typeof $ == 'undefined') ?  [] : " +
```
Comment (aeng) [inline functional-test/src/main/java/org/zanata/page/AbstractPage.java]: We actually writing scripts to test scripts. :scream:

File: functional-test/src/main/java/org/zanata/page/AbstractPage.java [hunk 2]
```diff
@@ -435,4 +371,203 @@ private void assertReady(WebElement targetElement) {
...
+        getExecutor().executeScript("arguments[0].blur()", element);
+        waitForPageSilence();
+    }
+
+    /* The system sometimes moves too fast for the Ajax pages, so provide a
+     * pause
+     */
+    public void slightPause() {
+        try {
+            Thread.sleep(500);
```
Comment (aeng) [inline functional-test/src/main/java/org/zanata/page/AbstractPage.java]: I assume the `500` timeout is from observation?

**Comment by aeng** [comment]
:boom: :+1:  :clap: :boom:

File: functional-test/src/main/java/org/zanata/page/AbstractPage.java (hunk 1 above)
Comment (djansen-redhat) [inline functional-test/src/main/java/org/zanata/page/AbstractPage.java]: It is somewhat unfortunate that the Ajax silencer doesn't always wait for the notifications.

File: functional-test/src/main/java/org/zanata/page/AbstractPage.java (hunk 2 above)
Comment (djansen-redhat) [inline functional-test/src/main/java/org/zanata/page/AbstractPage.java]: Yes, an old artifact of dodgy development. It's not a new addition, but moved from elsewhere. The sooner I can remove it entirely, the better.

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested by our friend, Jenny Kins.

**Comment by djansen-redhat** [comment]
Merged 4ca3c35f61c69073e8896bef37444b312111cfa6

**Pull request closed without merging**